backend/src/db/migrate/store.ts

```diff
@@ -36,8 +36,8 @@ class Store {
 
       // You have to wait for the schema to install, else the constraints will not be present.
       // This is a type error of the library
-      // eslint-disable-next-line @typescript-eslint/await-thenable
-      await getNeode().schema.install()
+
+      getNeode().schema.install()
```

Comment:
you must wait it - it doesnt work properly if you don't - I tested it - see comment above

Reply:
That's so strange. So we must *not* `await` here? Is that right?

Reply:
The types are broken. It is a Promise thats returned. This library is just a nightmare.